_authors/osci_2023/RobertKilkenny.markdown

```diff
@@ -0,0 +1,6 @@
+
+short_name: Robert
+name: Robert Kilkenny
+github: RobertKilkenny
```

Comment:
```suggestion
---
short_name: Robert
name: Robert Kilkenny
github: RobertKilkenny
---
```

The three dashes are important for parsing the "front matter" of this markdown file. See https://jekyllrb.com/docs/front-matter/